src/page-objects/administration/LandingPageDetail.ts

```diff
@@ -0,0 +1,65 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class LandingPageDetail implements PageObject {
```

Comment:
Take care here. 

We have here 2 pages:

1. Create page with URL:  `#/sw/category/landingPage/create/base`;`
2. LandingPageDetails from a specific landing page with URL: `#/sw/category/landingPage/<UUIDOfLandingPage/base`;`

Depends on what you need and what you want to offer to the community, I would recommend to have 2 pageObject classes:

1. LandingPageCreate with `#/sw/category/landingPage/create/base`;`
2. LandingPageDetail with `#/sw/category/landingPage/<UUIDOfLandingPage/base`;`

What page properties which page object has depends on you :)